contracts/GaugeController.sol

```diff
@@ -215,7 +273,6 @@ contract GaugeController is IGaugeController, Ownable {
      function setGaugeReward(IGaugeReward _gaugeReward) external onlyOwner {
         require(address(_gaugeReward) != address(0), "GC/GaugeReward-not-zero-address");
         gaugeReward = _gaugeReward;
-
         emit GaugeRewardSet(_gaugeReward);
```

Comment:
We could pass the `msg.sender` here.

Reply:
It's the owner, so didn't really see a need. Maybe if it's possible from multiple accounts in the future?